### ISSUE TITLE
Mobile friendly feature/app

### ISSUE THREAD

**Issue by JoraKasapi** [JoraKasapi/TheAlbanianHouse#9]
Sonia and I were thinking that what could be useful while checking the different pins/buildings would be to have your location on the map too. This would bring a better sense of orientation for tourist or  simply people using our map. Can it be done without tracking the person?

**Comment by rskikuli** [comment]
I think this is whole new project... Needs insight from @sidorelauku @kominoshja

**Comment by AnXh3L0** [comment]
Yes, this is quite possible and can be done client-side without tracking the person using an HTML5 API for geolocation and the current map. Needs some work but yes, it's possible.
